rules/RES-005.md

```diff
@@ -0,0 +1,19 @@
+**Rule ID:** RES-005
+
+**Description:** Resource attribute `service.name` must be present
+
+**Rationale:** `service.name` represents the logical name of the service, and it is critical for service identification. This attribute is required by [OpenTelemetry Semantic Conventions for Resources](https://opentelemetry.io/docs/specs/semconv/resource/#service). Ideally, it must be the same for all instances of horizontally scaled services.
+
+**Target:** Resource
+
+**Criteria:** Resource attributes must contain a `service.name` key with a non-empty string value. The attribute must not be null, undefined, or an empty string.
```

Comment:
We briefly talked about this over slack, I think we should also trigger this rule if the default value (unknown service) is used.